shell/components/CopyToClipboardText.vue

```diff
@@ -12,18 +12,38 @@ export default {
     }
   },
 
+  data() {
+    return { copied: false };
+  },
+
   methods: {
     clicked(event) {
-      event.preventDefault();
-      this.$copyText(this.text);
+      if (!this.copied) {
+        event.preventDefault();
+        this.$copyText(this.text);
+        this.copied = true;
+
+        let t = event.target;
```

Comment:
Using `:class="{'plain': plain, 'copied': copied }"` for the anchor would remove the need to manually add/remove the class in code